scripts/avatars/emerg_wds.py

```diff
@@ -0,0 +1,26 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+
+def main():
+    techops = GreatApeSafe(r.badger_wallets.techops_multisig)
+
+    # avatars
+    aura_avatar = techops.contract(r.avatars.aura)
+    convex_avatar = techops.contract(r.avatars.convex)
+
+    # withdraws
+    vanilla_aura_pids = aura_avatar.getPids()
+    if len(vanilla_aura_pids) > 0:
+        aura_avatar.withdrawAll()
+
+    private_pids = convex_avatar.getPrivateVaultPids()
+    if len(private_pids) > 0:
+        for pid in private_pids:
+            convex_avatar.withdrawFromPrivateVault(pid)
```

Comment:
getting this after
```
>>> run('scripts/avatars/trops_avatars_seeding.py', 'seed')
>>> run('scripts/avatars/emerg_wds.py')
```
```
TransparentUpgradeableProxy.withdrawFromPrivateVault confirmed (typed error: 0x37e8cabe00000000000000000000000091bf35e7441aeb10c007a4b7a1afa049e5420f8880e2a73f275e76d9dd4e0994689f44391e540b0a1d85aa08ade135aa43d080b3)   Block: 16942433   Gas used: 44876 (0.67%)
```
call trace shows an problem on `AuraAvatarMultiToken._withdraw`

Reply:
the problem its more in the private vault, since its locked for the min lock time (aprox 7d), should be reverting in an immediate sequence testing. push an advance of time to circumvent this in 2f57c7de542c945af19ecd054c372151629a8dea